app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    result = []
+    for person in people:
+        obj = Person(person["name"], person["age"])
+        Person.people[person["name"]] = obj
```

Comment:
You can use list comprehension here 
```suggestion
    result = [Person(person["name"], person["age"]) for person in people]
```